src/App.tsx

```diff
@@ -14,11 +15,145 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+type State = {
+  listIsVisible: boolean,
+  copyOfGoods: string[],
+  initialGoods: string[],
+  listIsReversed: boolean,
+  sortBy: string,
+};
+
+// eslint-disable-next-line react/prefer-stateless-function
+class App extends React.Component<{}, State> {
+  state = {
+    listIsVisible: false,
+    // eslint-disable-next-line react/no-unused-state
+    copyOfGoods: goodsFromServer,
+    listIsReversed: false,
+    initialGoods: goodsFromServer,
+    sortBy: '',
```

Comment:
do you need copyOfGoods and initialGoods?
I believe you can use only one array for goods in state